SerialPrograms/Source/PokemonSV/Programs/General/PokemonSV_StatsReset.cpp

```diff
@@ -166,27 +167,46 @@ void StatsReset::program(SingleSwitchProgramEnvironment& env, BotBaseContext& co
                 env.update_stats();
                 env.console.log("Unable to read ball quantity.", COLOR_RED);
             }
-            //Throw ball/Wild pokemon's turn/wait for catch animation
+            //Throw ball
             pbf_mash_button(context, BUTTON_A, 150);
-            stats.balls++;
-            env.update_stats();
-            //pbf_wait(context, 4000);
-            pbf_mash_button(context, BUTTON_B, 1000);
-            context.wait_for_all_requests();
 
-            AdvanceDialogWatcher summary(COLOR_MAGENTA);
-            int ret2 = wait_until(
+            //Check for battle menu - if found after a second then assume caught in a goldfish bounce loop
```

Comment:
Should it be 4 or 1 second ?

Reply:
I think the comment is wrong. 4 is correct since it takes a lot longer than 4 seconds from throwing a ball to returning to the battle menu after breaking out. Whereas if you are stuck, you see it a lot faster - but possibly longer than 1 second depending on text speed and how laggy the game is.

Reply:
I'll merge and fix the comment in a followup.